### PR TITLE
Add new glx stations to Signs UI

### PR DESCRIPTION
**Pull request by PaulJKim**
#### Summary of changes
**Asana Ticket:** [Updates to signs ui](https://app.asana.com/0/1199504266721641/1200959142579183/f)

This PR adds options to the Signs UI for the GLX stations that are planned to be opened in the next couple months. This is a precursor to updates to realtime signs as well as field testing of these signs that were added.

Screen Shot 2022-01-24 at 11 21 23 AM

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on coverage statistics)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes (compare on Splunk: [staging](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Dsigns-ui-dev%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840769.3874970) vs. [prod](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Dsigns-ui-prod%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840745.3874956))

### CODE REVIEW

**Review by losvedir — COMMENTED** [review]
I think you'll also want to update `arincToRealtimeIdMap` in `mbta.ts`, which maps the ARINC id/zone to some arbitrarily determined ID used internally by realtime_signs.

You might also want to enable `custom: true` on the signs, since putting some custom text on there in dev is one of the common ways that we test that realtime-signs and signs-ui are correctly integrated. Having two options also allows the frontend to toggle between the two, and fiddling with the values is what should actually end up persisting these new signs to the config JSON in S3.

**Comment by PaulJKim** [comment]
That makes sense for adding the custom option.

So these mappings correspond to the ids in `priv/signs.json` over on Realtime Signs, right? After we set the id in the `mbta.ts` file here, we just need to configure Realtime signs to have those id's in that config file?

File: assets/js/mbta.ts
Comment (PaulJKim) [inline assets/js/mbta.ts:2829]: Updated this one because there was an Id colliding with the new GLEC-m zone on the new train stop, and to more accurately map s to south.

**Comment by losvedir** [comment]
> So these mappings correspond to the ids in `priv/signs.json` over on Realtime Signs, right? After we set the id in the `mbta.ts` file here, we just need to configure Realtime signs to have those id's in that config file?

Yep, exactly.

File: assets/js/mbta.ts
Comment (PaulJKim) [reply assets/js/mbta.ts:2824]: Will definitely double check with Andrew on that. I figured that the `GLEC` zones belonged to the new train stop and the `SLEC` zone belonged to bus, but I wasn't sure if we wanted to still keep the old busway signs considering there are currently still buses running out of the old Lechmere station.

I guess that mainly begs the question of what exactly the rollout plan is for the new zones. Although these changes don't need to go to prod until the station officially opens, I wonder if, when that happens, it will be an immediate and full cutover to the new train station/busway or would there be any reason to continue running buses out of old lechmere for some time when the new station opens?

Comment (losvedir) [reply assets/js/mbta.ts:2824]: Oh, I was under the impression the PA/ESS signs at the old Lechmere were offline. Can you check with Dan Salomon about that? And I think the new signs are up and running. If that's the case, I think you can do a hard cutover now, and we can just set them to "Off".

Comment (PaulJKim) [reply assets/js/mbta.ts]: Ah I see, I spoke to Dan and the signs are still powered on but because they're cut off from the network, there's nothing syncing them to keep them from drifting so I think the times are off by several hours. So it does seem like it'll be fine to completely remove them. Probably also will be worth reaching out to New Era Tech to have those signs turned off since they're not doing anything helpful.

File: assets/js/mbta.ts
Comment (PaulJKim) [inline assets/js/mbta.ts:1995]: @losvedir I added some labeling to GUNS and GLEC to help reflect the reality of the stations and with the aim of helping avoid confusion for users regarding the weirdness with cardinal directions along the GLX. Let me know if you have any thoughts or if you think this makes sense!

File: assets/js/mbta.ts
Comment (digitalcora) [inline assets/js/mbta.ts:3176]: Doesn't this have to match the realtime_signs config? So either this should be `east` and `west`, or realtime_signs should be `north` and `south`.

File: assets/js/mbta.ts
Comment (digitalcora) [inline assets/js/mbta.ts:2815]: Should this just be `Lechmere`? It appears the other busway sign definitions use the plain station name, like `Nubian` or `Harvard`.

File: assets/js/mbta.ts
Comment (digitalcora) [reply assets/js/mbta.ts]: Ideally station signage would indicate the platforms as e.g. "Track 1" and "Track 2", then we would have official names for them. @jfabi do you know if that's the case?

File: assets/js/mbta.ts
Comment (PaulJKim) [reply assets/js/mbta.ts:3176]: oops yes, I had made those but forgot to push them. Thanks for calling that out!

File: assets/js/mbta.ts
Comment (PaulJKim) [reply assets/js/mbta.ts:2815]: Oh yeah, we should be able to do that since we deleted the old Lechmere busway!

File: assets/js/mbta.ts
Comment (jfabi) [reply assets/js/mbta.ts]: Yes, **Union Square** already has the identifiers "Track 1" and "Track 2" installed as follows (photo taken from Prospect Street):

DSC_0427

**At Lechmere**, the platforms are not yet signed up, but when they are installed, they will read "(D) Union Square / (E) College Ave" (on the same track as Track 1 at Union Square), and "Copley & West" (on the same track as Track 2 at Union Square).

Comment (PaulJKim) [reply assets/js/mbta.ts]: Thank you for the insight here @jfabi and @digitalcora , I was stuck on what to name these for a bit since they're both "westbound" tracks. Just pushed a change to reflect the correct track identifiers here and if it looks good to you @digitalcora , I will go ahead and make the corresponding change on the realtime signs end!

File: assets/js/mbta.ts
Comment (digitalcora) [inline assets/js/mbta.ts:2027]: Does this fit into the UI? We might want something shorter. (I assume it's fine if it's `Union Sq` for now, until the rest of GLX opens)

File: assets/js/mbta.ts
Comment (jfabi) [reply assets/js/mbta.ts]: Just for the record (because I was originally going off of blueprints without thinking), it should be "(D) Union Square / (E) Medford/Tufts", since College Ave station has since been renamed Medford/Tufts. (Aside: I wonder how the slashes will work there.)

This shouldn't affect this pull request as you've dropped "College Ave" entirely from the label.

**Review by digitalcora — APPROVED** [review]
Seems ok to me to start testing in dev! Of course, if you can wait until tomorrow morning, @losvedir could also give it a final once-over.

**Comment by PaulJKim** [comment]
Thanks for the review @digitalcora and sounds like a good plan to me, will wait for final approval from @losvedir!

File: assets/js/mbta.ts
Comment (losvedir) [inline assets/js/mbta.ts:3184]: Did you talk to Dan Salomon about these changes? These IDs come from TransitWay Engine which he maintains.

**Review by losvedir — APPROVED** [review]
🚀 🎉

File: assets/js/mbta.ts
Comment (PaulJKim) [reply assets/js/mbta.ts:3184]: Not yet - will reach out and make sure we're on the same page about the ids!

File: assets/js/mbta.ts
Comment (dks-mbta) [inline assets/js/mbta.ts:2046]: Should we have `label: 'Busway',` ?

File: assets/js/mbta.ts
Comment (dks-mbta) [inline assets/js/mbta.ts:2044]: For consistency, this should be under SLEC rather than GLEC.

File: assets/js/mbta.ts
Comment (dks-mbta) [inline assets/js/mbta.ts:2783]: Should this include `label: 'Busway',` ?

File: assets/js/mbta.ts
Comment (dks-mbta) [reply assets/js/mbta.ts:3184]: Ultimately this has to agree with ARINC's zone assignments. If the plan is to have the busway be SLEC m, this is fine with me, and I'll configure the Transitway Engine to agree with this. Will there also be sign(s) in the GLEC m zone for rail predictions?

Are these configured as `auto: false,` and `headway: false,` to lock off the signs because revenue service hasn't started?

Comment (PaulJKim) [reply assets/js/mbta.ts:3184]: That's correct - SLEC-m is for buses and there is also GLEC-m for rail.

And yep, the plan currently is to follow up and unlock those signs for `headway` and `auto` mode once revenue service begins

File: assets/js/mbta.ts
Comment (PaulJKim) [reply assets/js/mbta.ts:2783]: Hmm yeah probably a good idea. Although the UI does have an overarching **Busway.** label at the top of this page, 'Mezzanine' is kind of ambiguous in this context so Busway is probably clearer than that

**Pull request merged**
